### PR TITLE
Use react optimize, fixes #255

### LINKED ISSUE #255: Consider using babel-react-optimize
With #248, we'll be updating to `react@15.0.1`. This means we should be able to take advantage of perf optimizations for free by using [babel-react-optimize](https://github.com/thejameskyle/babel-react-optimize).

### PR DESCRIPTION
**Pull request by victorporof**
https://github.com/mozilla/tofino/issues/255

@Mossop, this involves a few build changes, mind having a look?

### CODE REVIEW

**Comment by victorporof** [comment]
See individual commits for a step by step explanation of what should change.

File: build/tasks.js [hunk 1]
```diff
@@ -30,45 +30,45 @@ export default {
   },
 
   async run(args = []) {
-    await this.build();
+    await this.build([...args, '--force']);
```
Comment (Mossop) [inline build/tasks.js]: This is fixing #357 and I think it's fine to do it like this for now. What might be nice in the future is to load build-config first, then diff it against the new build-config and in that case turn force on.

**Comment by Mossop** [comment]
LGTM

File: build/tasks.js (hunk 1 above)
Comment (victorporof) [inline build/tasks.js]: Sounds good!

Comment (victorporof) [inline build/tasks.js]: Maybe re-purpose that issue to follow this, or close that and file a new one instead?

Comment (Mossop) [inline build/tasks.js]: I've opened #528

**Pull request merged**